File: src/main/java/gargoyle/ct/ui/impl/control/CTShowingFrame.java
package gargoyle.ct.ui.impl.control;

import javax.swing.*;
import java.awt.*;

public class CTShowingFrame extends JFrame {
    private static final long serialVersionUID = 6336577268777238958L;

    public CTShowingFrame() throws HeadlessException {
    }

    @Override
    public boolean isShowing() {
        return true;
    }
}

File: src/main/kotlin/gargoyle/ct/ui/impl/control/CTShowingFrame.kt
package gargoyle.ct.ui.impl.control

import java.io.Serial
import javax.swing.JFrame

class CTShowingFrame : JFrame() {
    override fun isShowing(): Boolean {
        return true
    }

    companion object {
        @Serial
        private val serialVersionUID = 6336577268777238958L
    }
}
